docs/iris/src/developers_guide/documenting/whats_new_contributions.rst

```diff
@@ -64,12 +119,15 @@ users.  To achieve this several categories may be used.
 *Incompatible Changes*
   A change that causes an incompatibility with prior versions of Iris.
 
-*Internal*
-  Changes to any internal or development related topics, such as testing,
-  environment dependencies etc
-
 *Deprecations*
   Deprecations of functionality.
 
+*Dependencies*
+  Additions, removals and version changes in Iris' package dependencies.
+
 *Documentation*
   Changes to documentation.
+
+*Internal*
+  Changes to any internal or development related topics, such as testing,
+  environment dependencies etc
```

Comment:
There should be a full stop after "etc".

Reply:
3138a01bbee3e9e2c67644ff9b4eae4eddde73aa